.gitignore

```diff
@@ -1,3 +1,4 @@
 /node_modules
 /vendor
 .env
+composer.lock
```

Comment:
The StackOverflow comment you read was wrong. Take a look at the same StackOverflow thread, but at the accepted answer, and you'll get a bit of a clue why. For more reasons, check out this link:

https://blog.engineyard.com/2014/composer-its-all-about-the-lock-file

So, please bring back `composer.lock` for this PR. :)